external/header.php

```diff
@@ -7,6 +7,26 @@
 
 include(dirname(__FILE__) . '/../xhprof_lib/config.php');
 
+function getExtensionName()
+{
+    if( extension_loaded( 'tideways' ) )
```

Comment:
I agree, that no visible coding standard can be found in there. But at least don't make it look too different from existing code 😉